Source/WebKit/NetworkProcess/webrtc/NetworkRTCMonitor.cpp

```diff
@@ -211,7 +211,7 @@ static UncheckedKeyHashMap<String, RTCNetwork> gatherNetworkMap()
 
         auto name = span(iterator->ifa_name);
         auto prefixString = address->second.rtcAddress().ToString();
-        auto networkKey = makeString(StringView { name }, "-"_s, prefixLength, "-"_s, StringView { makeSpan(prefixString) });
+        auto networkKey = makeString(StringView { name }, "-"_s, prefixLength, "-"_s, StringView { prefixString });
```

Comment:
Why do we need to construct a `StringView`? I think `makeString` knows how to append a `span` and it's more efficient?

Reply:
/Volumes/Work/WebKit/OpenSource/WebKitBuild/Debug/usr/local/include/wtf/text/MakeString.h:93:38: error: implicit instantiation of undefined template 'WTF::StringTypeAdapter<std::string>'

Reply:
We could go with this though:
```
auto networkKey = makeString(name, "-"_s, prefixLength, "-"_s, std::span { prefixString });
```